src/hooks/screen/useChatInfoHook.ts

```diff
@@ -59,8 +59,8 @@ function useChatInfoScreenHook(): UseAnonymousChatInfoScreenHook {
     return navigation.push('OtherProfile', {
       data: {
         user_id: myUserId?.myProfile?.user_id,
-        other_id: item?.user_id,
-        username: item?.user?.name
+        other_id: item?.user_id || item?.userId,
+        username: item?.user?.name || item?.user?.username
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for handling potential null or undefined values. By providing fallbacks, you're ensuring that the application doesn't crash due to unexpected null or undefined values. However, it's important to consider what happens when these fallbacks are used. If `item?.userId` or `item?.user?.username` are not valid or meaningful values in your application context, this could lead to unintended behavior.
```diff
-        other_id: item?.user_id,
-        username: item?.user?.name
+        other_id: item?.user_id || item?.userId,
+        username: item?.user?.name || item?.user?.username
```
If `item?.userId` and `item?.user?.username` are indeed valid and expected properties, then these changes are appropriate.


<!-- This is an auto-generated comment by OSS CodeRabbit -->